whelk-core/src/main/resources/ext/marcframe.json

```diff
@@ -8311,8 +8445,11 @@
       },
       "$a": {"about": "_:contentType", "property": "label", "infer": true},
       "$b": {"about": "_:contentType", "property": "code"},
+      "$0": {"about": "_:contentType", "property": "marc:recordControlNumber"},
```

Comment:
Adding these ($0 and $6) to a potentially linked resource requires the presence of these on the linked resource. Is that the intent?

Reply:
Having thought and discussed it, this isn't worse than the already mapped $8. If these ever crop up in data, they're a signal to investigate and restructure.